src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/ModelBindingHelper.cs

```diff
@@ -20,9 +20,9 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding
     public static class ModelBindingHelper
     {
         /// <summary>
-        /// Updates the specified <paramref name="model"/> instance using the specified <paramref name="modelBinder"/>
-        /// and the specified <paramref name="valueProvider"/> and executes validation using the specified
-        /// <paramref name="validatorProvider"/>.
+        /// Updates the specified <paramref name="model"/> instance using the specified
+        /// <paramref name="modelBinderFactory"/> and the specified <paramref name="valueProvider"/> and executes 
+        /// validation using the specified <paramref name="validatorProvider"/>.
```

Comment:
If we meant for this class to be `public` and in a prominent namespace, this is a significant API change and does not support the previous scenarios in a straightforward fashion.